Sources/macro.js

```diff
@@ -7,6 +7,52 @@ export function getCurrentGlobalMTime() {
 }
 
 // ----------------------------------------------------------------------------
+// Loggins function calls
+// ----------------------------------------------------------------------------
+
+const fakeConsole = {};
+function noOp() {}
+
+const consoleMethods = ['log', 'debug', 'info', 'warn', 'error', 'time', 'timeEnd', 'group', 'groupEnd'];
+consoleMethods.forEach(methodName => (fakeConsole[methodName] = noOp));
+
+global.console = window.console || fakeConsole;
```

Comment:
I feel like we need to do this for ParaViewWeb (and ERS) - is there a good place?